README.md

```diff
@@ -22,6 +22,13 @@ and OpenCV 3 (instructions for
 [Linux](http://docs.opencv.org/3.2.0/d7/d9f/tutorial_linux_install.html))
 installed in order for `frc-livescore` to work.
 
+### Installation Through pip
+```bash
+ $ pip install numpy
+ $ pip install opencv-python
+ $ sudo pip install pytesseract
```

Comment:
Both `pytesseract` and `numpy` are in the `requirements.txt`. Doing `sudo pip install -r requirements.txt` should be enough to install those two. Could you change to that? Also if `opencv-python` works, then it would be good to add that to the `requirements.txt` and just make it all one easy command.